triton/access_config.go

```diff
@@ -14,10 +14,10 @@ import (
 
 // AccessConfig is for common configuration related to Triton access
 type AccessConfig struct {
-	Endpoint string `mapstructure:"sdc_url"`
-	Account  string `mapstructure:"sdc_account"`
-	KeyID    string `mapstructure:"sdc_key_id"`
-	KeyPath  string `mapstructure:"sdc_key_path"`
+	Endpoint string `mapstructure:"triton_url"`
+	Account  string `mapstructure:"triton_account"`
+	KeyID    string `mapstructure:"triton_key_id"`
+	KeyPath  string `mapstructure:"triton_key_path"`
```

Comment:
It appears to be `key_material` with the actual private key contents in the Terraform provider: https://github.com/hashicorp/terraform/blob/master/builtin/providers/triton/provider.go#L32. Which one is preferable?